src/core_atmosphere/Registry.xml

```diff
@@ -3566,11 +3566,11 @@
 
                 <var name="taod5502d" type="real" dimensions="nCells Time" units="unitless"
                      description="total aerosol optical depth at 550 nm from the Thompson cloud microphysics"
-                     packages="mp_thompson_aers_in"/>
+                     packages="mp_thompson_aers_in;mp_tempo_in"/>
```

Comment:
I think this variable and the one below should be allocated in this package: `tempo_aerosolaware_in`

Reply:
changed